app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -186,8 +209,13 @@ class GithubWebhook extends RequestHandler<Body> {
     ]));
     return batch.responses
         .map((Response response) => response.searchBuilds)
-        .expand((SearchBuildsResponse response) => response.builds)
-        .toList();
+        .expand((SearchBuildsResponse response) {
+      if (response.builds != null) {
+        return response.builds;
+      } else {
+        return <Build>[];
+      }
```

Comment:
Could be simplified to `return response.builds ?? <Build>[];`

Reply:
Done!